parm/ioda/bufr_atms.yaml

```diff
@@ -0,0 +1,176 @@
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "$(BUFR_in)"
+
+      exports:
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+              second: "*/SECO"
+
+          latitude:
+            query: "*/CLATH"
+
+          longitude:
+            query: "*/CLONH"
+
+          satelliteIdentifier:
+            query: "*/SAID"
+
+          satelliteInstrument:
+            query: "*/SIID"
+
+          # The type should be integer.
+          # However, UFO/CRTM requred this variable to be float
+          fieldOfViewNumber:
+            query: "*/FOVN"
+            type: float
```

Comment:
Curious.  When will ufo and/or crtm be updated to allow FOVN to be an integer as it should be?